### PR TITLE
Swift Package Manager Support

### PR DESCRIPTION
**Pull request by scannillo**
### Summary of changes

 - Add Package.swift
 - Add SPMTest app
 - Add rake release task for building SPMTest app
 - Remove `Classes` dir & add `Public` dir
 - Relink & embed PopupBridge framework to Demo app (must have forgotten to commit in previous PR)

 ### Checklist

 - [X] Added a changelog entry

### Authors
- @scannillo

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,6 @@
 language: objective-c
 osx_image: xcode12.3
 
-branches:
-  only:
-  - master
-
```
Comment (scannillo) [inline .travis.yml:7]: Allow Travis CI to run on all PRs (not just those against master)

File: Demo/Demo.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -14,6 +14,8 @@
 		806D559125AFD735005ED326 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 806D559025AFD735005ED326 /* Assets.xcassets */; };
 		806D559725AFD735005ED326 /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 806D559625AFD735005ED326 /* main.m */; };
 		806D55B725AFD877005ED326 /* PopupBridge_DemoUITests.m in Sources */ = {isa = PBXBuildFile; fileRef = 806D55B625AFD877005ED326 /* PopupBridge_DemoUITests.m */; };
+		806F556025B1400A001754AF /* PopupBridge.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 806F555F25B1400A001754AF /* PopupBridge.framework */; };
+		806F556125B1400A001754AF /* PopupBridge.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 806F555F25B1400A001754AF /* PopupBridge.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```
Comment (scannillo) [inline Demo/Demo.xcodeproj/project.pbxproj:18]: Relink & embed PopupBridge framework to Demo app (must have forgotten to commit in previous PR ? )

File: PopupBridge.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -480,14 +504,16 @@
...
+				INFOPLIST_FILE = "$(SRCROOT)/Sources/PopupBridge/PopupBridge-Framework-Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 10.1;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				MTL_ENABLE_DEBUG_INFO = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = com.braintreepayments.PopupBridge;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
+				SUPPORTS_MACCATALYST = NO;
```
Comment (scannillo) [inline PopupBridge.xcodeproj/project.pbxproj:515]: This SDK does not support Mac Catalyst

File: PopupBridge.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -511,14 +537,16 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				GCC_NO_COMMON_BLOCKS = YES;
-				INFOPLIST_FILE = "$(SRCROOT)/PopupBridge/PopupBridge-Framework-Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/Sources/PopupBridge/PopupBridge-Framework-Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 10.1;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (scannillo) [inline PopupBridge.xcodeproj/project.pbxproj:542]: The Framework should be supporting iOS 9+ 🙃

**Review by sestevens — APPROVED** [review]
Looks good!

File: .travis.yml [hunk 5]
```diff
@@ -2,9 +2,9 @@ language: objective-c
 osx_image: xcode12.3
 
 branches:
-  only:
-  - master
-  
+  only: 
+    - master
```
Comment (scannillo) [inline .travis.yml:6]:
```suggestion
  only:
  - master
```

File: .travis.yml (hunk 1 above)
Comment (scannillo) [reply .travis.yml]: This didn't do what I thought it did - reverted.

**Review by sshropshire — APPROVED** [review]

**Pull request merged**